uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl

```diff
@@ -798,7 +799,11 @@
             <xsl:copy-of select="."/>
         </xsl:for-each>
     </head>
-    <body class="up dashboard portal fl-theme-mist detachedHeader">
+    <body>
+        <xsl:attribute name="class">
+            <xsl:text>up dashboard portal fl-theme-mist detachedHeader</xsl:text>
```

Comment:
Is `fl-theme-mist` needed at all any longer?

Reply:
Not sure for me !

Reply:
There was another PR earlier to remove all Fluid classes, and I think that's still something we want to do.  (The reason we didn't merge it is that many of them were used in jQuery selectors and in custom style rules... so we need to clean that stuff up.)

Reply:
#1051 was the PR